redbot/cogs/mutes/mutes.py

```diff
@@ -639,20 +674,24 @@ async def _check_for_mute_role(self, ctx: commands.Context) -> bool:
         This explains to the user whether or not mutes are setup correctly for
         automatic unmutes.
         """
+        command_1 = f"{ctx.clean_prefix}muteset role"
+        command_2 = f"{ctx.clean_prefix}muteset makerole"
+        msg = _(
+            "This server does not have a mute role setup, "
```

Comment:
You already have one space below. Judging by next line, I'm also assuming you wanted to have a dot here, not a comma.
```suggestion
            "This server does not have a mute role setup."
```